templates/preprocessing-worker.toml.j2

```diff
@@ -13,3 +13,29 @@ maxConcurrentSessions = 1
 
 [bagit]
 checksumAlgorithm = "md5"
+
+[ais]
+listen = "127.0.0.1:9035"
+workingDir = "/tmp"
+
+[ais.temporal]
+address = "127.0.0.1:7233"
+namespace = "default"
+taskQueue = "ais"
+workflowName = "ais"
+
+[ais.worker]
+maxConcurrentSessions = 1
+
+[ais.amss]
+url = "http://127.0.0.1:8000"
+user = "test"
+key = "test"
+
+[ais.bucket]
```

Comment:
Instead of hardcoded values, we should use template vars, as we do here: https://github.com/artefactual-labs/ansible-enduro-temporal/blob/dev/add-preprocessing/templates/enduro.toml.j2#L140

(for both, ss and minio bucket config)

Reply:
Not sure why is the section you linked inside a for loop over `enduro_sfa_pipelines`. I guess this expects a single pipeline configuration in that array. For the preprocessing template I've used `{% set pipeline = enduro_sfa_pipelines[0] %}` instead of the for loop.

You mentioned once to use the same AMSS URL schema from legacy Enduro here, including the user and the API key, but that's not implemented yet, so we are expecting two new vars for them for now.